hw/mcu/dialog/da1469x/syscfg.yml

```diff
@@ -113,6 +113,16 @@ syscfg.defs:
             data before writing, i.e. each such transfer will be split into
             chunks of this size. Buffer is created on stack.
         value: 128
+    QSPI_BURSTCMDA_INIT_VAL:
```

Comment:
since this is specific to this particular MCU family amd not QSPI in general I'd call these settings e.g. `MCU_QSPIC_BURSTCMDA...`

Reply:
Done.